### DISCUSSION TITLE
Add User Profile

### DISCUSSION THREAD

**Discussion by darkjoka** [JhohannesK/TipoHiZ#27] [Ideas]
**Is your feature request related to a problem? Please describe.**
We currently only have username and email as information on users

**Describe the solution you'd like**
I propose we had a profile schema which should at least contain some user stats like typing tests completed, bio, leaderboard ranking etc

**Comment by JhohannesK** [comment]
We could start working on this

**Comment by darkjoka** [comment]
okay what data would be needed

**Comment by JhohannesK** [comment]
as you listed.
* Highest WPM achieved
* leaderboard ranking
* bio [optional]

**Comment by darkjoka** [comment]
okay we can start with that for now
